packages/osd-utils/src/index.ts

```diff
@@ -30,4 +30,4 @@
 
 export * from './package_json';
 export * from './path';
-export * from './repo_root';
+export { REPO_ROOT, REPO_ROOT_8_3, UPSTREAM_BRANCH } from '@osd/cross-platform';
```

Comment:
`UPSTREAM_BRANCH` existed. `REPO_ROOT_8_3` is the only new one and I figured since `@osd/dev-utils re-exports everything from `@osd/util`, if someone sees `REPO_ROOT` imported from there and decides to use `REPO_ROOT_8_3` from the same import, they wouldn't be inconvenienced; my hope is that no one will ever need to use this but you are right that it needn't be exported from here.

Reply:
that explanation makes sense!